setup.py

```diff
@@ -215,6 +215,9 @@ def read(*rnames):
 #   }
 }
 
+if 'RADICAL_DEBUG' in os.environ:
+    setup_args['install_requires'].append('setproctitle')
+
```

Comment:
This is actually for both, reason being that we'll use the same things on the module site soon, too.  Otherwise we can add it to the bootstrapper -- setup.py itself does not make a distinction between the deployment modes.  We could do that, but I would not think the added complexity is worth it.  Maybe if that saves bootstrapping logic?  dunno...